tensorflow_serving/tools/docker/Dockerfile.devel

```diff
@@ -9,6 +9,7 @@ RUN apt-get update && apt-get install -y \
         libfreetype6-dev \
         libpng12-dev \
         libzmq3-dev \
+        locate \
```

Comment:
Thanks for the fix!

Let's change this to `mlocate` since I think it's recommended and it will be consistent with TF.